### ISSUE TITLE
Client sent an HTTP request to an HTTPS server. when visiting without port in url

### ISSUE THREAD

**Issue by mezoology** [linuxserver/docker-pydio#45]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

If you are new to Docker or this application our issue tracker is **ONLY** used for reporting bugs or requesting features. Please use [our discord server](https://discord.gg/YWrKVTn) for general support.

------------------------------

## Expected Behavior

proceed to complete the installation process, setup DB and users.
## Current Behavior

it gives the error : Client sent an HTTP request to an HTTPS server
## Steps to Reproduce

1. on openlitespeed, the docker exposed ports as 9090:80
2. i did the reverse proxy as this [guide](https://openlitespeed.org/kb/reverse-proxy-basics/) suggests
3.i put the following in /usr/local/lsws/conf/httpd_config.conf
`extprocessor docker {
  type                    proxy
  address                 127.0.0.1:9090
  maxConns                100
  pcKeepAliveTimeout      60
  initTimeout             60
  retryTimeout            0
  respBuffer              0
}`
4. then added the following to Rewrite Rules in subdomain.domain.com
`RewriteCond %{HTTPS} !=on
RewriteRule ^ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
REWRITERULE ^(.*)$ http://docker/$1 [P]`

## Environment
**OS:**      ubuntu 18.04
**CPU architecture:** x86_64
**How docker service was installed:**     portainer 

my setup is cyperpanel with openlitespeed, i used the openlitespeed reverse proxy as mentioned above
## Command used to create docker container (run/create/compose/screenshot)

https://imgur.com/a/XHpIuwQ
## Docker logs

`
Portainer

    Home
    local
    Dashboard
    App Templates
    Stacks
    Containers
    Images
    Networks
    Volumes
    Events
    Host
    Settings
    Extensions
    Users
    Endpoints
    Registries
    Settings

Portainer 1.24.0
Container logs admin
Portainer support
Containers > pydio-cells > Logs
my account log out
Log viewer settings
Auto-refresh logs
Wrap lines
Display timestamps
Fetch
Search
Lines
Actions

      

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.

[s6-init] ensuring user provided files have correct perms...exited 0.

[fix-attrs.d] applying ownership & permissions fixes...

[fix-attrs.d] done.

[cont-init.d] executing container initialization scripts...

[cont-init.d] 01-envfile: executing... 

[cont-init.d] 01-envfile: exited 0.

[cont-init.d] 10-adduser: executing... 

-------------------------------------

          _         ()

         | |  ___   _    __

         | | / __| | |  /  \ 

         | | \__ \ | | | () |

         |_| |___/ |_|  \__/

Brought to you by linuxserver.io

-------------------------------------

To support LSIO projects visit:

https://www.linuxserver.io/donate/

-------------------------------------

GID/UID

-------------------------------------

User uid:    1000

User gid:    1000

-------------------------------------

[cont-init.d] 10-adduser: exited 0.

[cont-init.d] 30-config: executing... 

generating self signed cert with SAN 0.0.0.0

Generating a RSA private key

..............+++++

................................+++++

writing new private key to '/config/keys/cert.key'

-----

[cont-init.d] 30-config: exited 0.

[cont-init.d] 99-custom-files: executing... 

[custom-init] no custom files found exiting...

[cont-init.d] 99-custom-files: exited 0.

[cont-init.d] done.

[services.d] starting services

[services.d] done.

[Configs] Upgrading: setting default config services/pydio.web.oauth/secret to ZcT3_P4Su3hoXszlQk2Wol9sLIB2k.IY

[Configs] successfully saved config after upgrade - Reloading from source

Welcome to Pydio Cells Home Edition installation

Pydio Cells Home Edition will be configured to run on this machine. Make sure to prepare the following data

 - IPs and ports for binding the webserver to outside world

 - MySQL 5.6+ (or MariaDB equivalent) server access

Pick your installation mode when you are ready.

Cannot find vaultKeyPath, creating new one /config/cells-vault-key

**************************************************************

     Warning! A keyring is not found on this machine,         

 	A Master Key has been created for cyphering secrets       

   It has been stored in /config/cells-vault-key               

   Please make sure to secure this file and update the configs

   with its new location, under the defaults/keyPath key.     

***************************************************************

2020-06-10T23:37:38.624Z	INFO	pydio.gateway.rest	started

2020-06-10T23:37:38.720Z	INFO	Registering /install/events for Polling

2020-06-10T23:37:38.720Z	INFO	pydio.rest.install	started

Activating privacy features... done.

https://0.0.0.0:8080

Installation Server is starting ...

 internal URL: https://0.0.0.0:8080

 external URL: https://xxx.yyyyy.com

2020-06-10T23:37:40.924Z	INFO	pydio.gateway.proxy	Restarting proxy	{"caddyfile": "\n\t\t https://0.0.0.0:8080 {\n\t\t\t root \"/config/static/install\"\n\t\t\t proxy /install 172.17.0.7:42397\n\t\t \t\n\t\t\ttls \"/config/keys/cert.crt\" \"/config/keys/cert.key\"\n\t\t }\n\t "}

2020-06-10T23:37:40.925Z	INFO	pydio.gateway.proxy	Restart done

Opening URL https://xxx.yyyy.com in your browser. Please copy/paste it if the browser is not on the same machine.

2020-06-11T08:39:31.910Z	INFO	pydio.rest.install	Received Install.Get request	{"response": "config:<dbConnectionType:\"tcp\" dbTCPHostname:\"localhost\" dbTCPPort:\"3306\" dbTCPName:\"cells\" dbTCPUser:\"root\" dbSocketFile:\"/tmp/mysql.sock\" dbSocketName:\"cells\" dbSocketUser:\"root\" dbManualDSN:\"root@tcp(localhost=3306)/cells\" dsName:\"pydiods1\" dsPort:\"38075\" dsFolder:\"/config/data\" externalMicro:\"43985\" externalGateway:\"42925\" externalWebsocket:\"42563\" externalFrontPlugins:\"42761\" externalDAV:\"40547\" externalWOPI:\"41019\" externalDex:\"36045\" externalDexID:\"cells-front\" externalDexSecret:\"AyhczrvLI2jTAaNqlpgFbrqu\" frontendLogin:\"admin\" frontendApplicationTitle:\"Pydio Cells\" frontendDefaultLanguage:\"en\" > "}

      
      
    

`

**Comment by aptalca** [comment]
1) This is the wrong repo, what you're looking for is https://github.com/linuxserver/docker-pydio-cells
2) Use https to connect to the interface, as it says in the error message (or rather, tell your reverse proxy to use https when connecting)
3) `on openlitespeed, the docker exposed ports as 9090:80` where is this coming from? The container is listening on port 8080 internally, and your portainer screenshot shows port mapping of 3034:8080
3) Wrap logs in triple backticks or upload to pastebin or the like

**Comment by mezoology** [comment]
- 1- yea my bad, didn't know there is a variation of it.
- 3- the 9090:80 and 3034:8080 was my mistake, I tried both but forgot witch in the screenshot.
- 4- sorry, I'm bad at this with several more things including coding.

-----------------------------------
I got this resolved by putting https://127.0.0.1:8080
```
extprocessor drive {
  type                    proxy
  address                 https://127.0.0.1:8080
  maxConns                100
  pcKeepAliveTimeout      60
  initTimeout             60
  retryTimeout            0
  respBuffer              0
}
```
